charts/server/templates/statefulset.yaml

```diff
@@ -72,14 +72,14 @@ spec:
               port: 8000
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
-          {{- if .Values.persistentVolume.enabled }}
           volumeMounts:
-          - name: data
-            mountPath: {{ .Values.persistentVolume.mountPath }}
-          {{- end }}
-          {{- if .Values.extraVolumeMounts }}
-            {{- toYaml .Values.extraVolumeMounts | nindent 12 }}
-          {{- end }}
+            {{- if .Values.persistentVolume.enabled }}
```

Comment:
just a small finding -> allow extraVolumeMounts without enable default persistentVolume

Reply:
Thanks! Let's outsource this one to a new issue/PR

Reply:
Actually it looks good in `main`: https://github.com/woodpecker-ci/helm/blob/6cfa8bd23c118c4f49379e8269a3c8076198c045/charts/server/templates/statefulset.yaml#L75-L79

Reply:
No it does not ...

if `Values.persistentVolume.enabled: false` and `extraVolumeMounts` is set, it will generate invalid yaml

Reply:
Ahh that's what you mean. I second that. I thought until now you were referring to `extraVolumeMounts` being conditioned on `persistentVolume.enabled`.